bebraven/platform
# Setup Canvas Sections in newly launched Canvas courses

When the New Program Launch page is used to launch a new program
and create Accelerator and LC Playbooks from their respective templates,
this commit adds functionality to setup the initial list of Canvas
Sections in those courses and associate them with every assignment
so that a staff member can use the Edit Assignment Dates UI to set the
due dates for each assignment on a section by section basis.

### Test Plan:
- Create two course templates your local database with real Canvas course
  IDs. E.g.
  `CourseTemplate.create! name: 'Accelerator Template', canvas_course_id: 48`
  `CourseTemplate.create! name: 'LC Playbook Template', canvas_course_id: 185`
- Setup 2 Cohort Schedules on the Salesforce Program object you're going to launch.
- Set the "Section Name in LMS Coach Course" field on that Salesforce Program.
- Use the "New Program Launch" page here: https://platformweb/course_management/launch
  to launch that program using your two templates.
- You should get a success email (or have it show in the server logs) after a minute or
  two and if you go into Canvas, both courses should be there with sections showing for every
  assignment on the Edit Assignment Dates page from the Assignments tab.
  Note: the Accelerator will have a section per Cohort Schedule and the LC Playbook will
  just have a catch-all section taken from the "Section Name in LMS Coach Course" field



### Description



### Motivation and Context




### Screenshots (if appropriate):



### Checklist:
